### ISSUE TITLE
Flickering LED lights 

### ISSUE THREAD

**Issue by Jaahn123** [reuterbal/photobooth#126]
**Describe the bug**
I build the circuit basically according to the scheme provided here. Instead of RGB LEDs I changed the code a bit to drive three white LED strips and change the brightness for each state. 
In each state there is flickering of the LED lights if the brightness is not set to 100 %.
I guess that the GPIO pins are software PWMed only and that the CPU of my Rpi 3B+ is at 100%. That would lead to some changes in the PWMing and therefore flickering of the LED lights dependent on the CPU usage. 

Has anyone observed this behaviour or colour changes of the RGB LED? 

**Expected behavior**

No flickering of LEDs

**Hardware (please complete the following information)**

 - Device [ Raspberry Pi 3B+]
 - Camera [Canon 6D]
 - GPIO: [Yes]

**Software (please complete the following information)**

 - OS [Raspbian Stretch

 

**Additional context**

For now I plan to use an Arduino to PWM the LED strips correctly. The information in which state the software currently is will be transmitted via the three GPIO pins as a combination of highs and lows.

**Comment by reuterbal** [comment]
Yes, I'm seeing similar issues for the RGBLED. After updating gpiozero and the Raspbian libraries to the latest versions, this problem worsened significantly. It helped to increase the PWM frequency for the LEDs drastically (I had to use at least 1kHz instead of the default 100Hz, with 3kHz it is bearable but still visible). Unfortunately, some upstream change seems to have caused this.

Generally, the software PWM used by gpiozero seems to be too slow and using an Arduino is definitely the better option.
